### PR TITLE
Adding Theano-PyMC, a fork of Theano for PyMC3

### PR DESCRIPTION
**Pull request by twiecki**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by twiecki** [comment]
@ericmjl is it OK if I add you as a maintainer next to me?

File: recipes/theano-pymc/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/T/Theano-PyMC/Theano-{{ version }}.tar.gz
+  sha256: bb566fa62f7923351299dfcb8947e42523057df65e81be444eb8ef0c6e602bf4
+  patches:
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install --no-deps --ignore-installed .
```
Comment (synapticarbors) [inline recipes/theano-pymc/meta.yaml:14]: It looks like there is a python entry_point. Please add it here:

https://github.com/pymc-devs/Theano-PyMC/blob/master/setup.py#L131

https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#python-entry-points

File: recipes/theano-pymc/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    - theano.scalar.tests
+    - theano.sparse
+    - theano.sparse.tests
+    - theano.tensor
+    - theano.tensor.nnet
+    - theano.tensor.nnet.tests
+    - theano.tensor.signal
+    - theano.tensor.signal.tests
+    - theano.tensor.tests
+    - theano.tests
```
Comment (synapticarbors) [inline recipes/theano-pymc/meta.yaml:53]: Add `commands` test for entry point:

https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#test-commands

File: recipes/theano-pymc/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,66 @@
+{% set version = "1.0.5" %}
+
+package:
+  name: theano-pymc
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/T/Theano-PyMC/Theano-{{ version }}.tar.gz
```
Comment (synapticarbors) [inline recipes/theano-pymc/meta.yaml]: It looks like the `url` is wrong. Should be

```
.../Theano-PyMC-{{ version }}.tar.gz
```

I think

Comment (twiecki) [reply recipes/theano-pymc/meta.yaml]: Fixed.

File: recipes/theano-pymc/meta.yaml (hunk 1 above)
Comment (twiecki) [reply recipes/theano-pymc/meta.yaml:14]: Sorry, not quite sure what I need to do here. Could you provide a bit more info?

File: recipes/theano-pymc/meta.yaml (hunk 2 above)
Comment (twiecki) [reply recipes/theano-pymc/meta.yaml:53]: Sorry, not quite sure what I need to do here. Could you provide a bit more info?

File: recipes/theano-pymc/meta.yaml (hunk 1 above)
Comment (synapticarbors) [reply recipes/theano-pymc/meta.yaml:14]:
```
build:
  number: 0
  entry_points:
    - theano-cache = bin.theano_cache:main
  script: {{ PYTHON }} -m pip install . -vv
```

File: recipes/theano-pymc/meta.yaml (hunk 2 above)
Comment (synapticarbors) [reply recipes/theano-pymc/meta.yaml:53]: And then if `theano-cache` supports a "help" mode:

```
test:
   import:
      - theano
      ...
   commands:
    - theano-cache --help
```

File: recipes/theano-pymc/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,66 @@
+{% set version = "1.0.5" %}
+
+package:
+  name: theano-pymc
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/T/Theano-PyMC/Theano-PyMC{{ version }}.tar.gz
```
Comment (synapticarbors) [inline recipes/theano-pymc/meta.yaml:8]: `Theano-PyMC-{{ version }}.tar.gz` (missing dash after "PyMC")

File: recipes/theano-pymc/meta.yaml (hunk 1 above)
Comment (twiecki) [reply recipes/theano-pymc/meta.yaml]: Done.

File: recipes/theano-pymc/meta.yaml (hunk 2 above)
Comment (twiecki) [reply recipes/theano-pymc/meta.yaml:55]: Done.

File: recipes/theano-pymc/meta.yaml (hunk 4 above)
Comment (twiecki) [reply recipes/theano-pymc/meta.yaml]: Done.

**Comment by twiecki** [comment]
Thanks for the quick and useful feedback @synapticarbors!

**Pull request merged**